src/main.rs

```diff
@@ -43,6 +46,18 @@ pub struct Args {
     /// Percentage of profit to pay in gas.
     #[arg(long)]
     pub bid_percentage: u64,
+
+    /// Private key for sending txs.
+    #[arg(long)]
+    pub executor_address: String,
+
+    /// Order type to use.
+    #[arg(long)]
+    pub order_type: String,
```

Comment:
I think you can directly take the enum, structopt should be able to parse on its own

Reply:
ah nice yea